tests/unit/s2n_client_cert_verify_test.c

```diff
@@ -42,6 +45,62 @@ static int test_sign(const struct s2n_pkey *priv_key, s2n_signature_algorithm si
     return S2N_SUCCESS;
 }
 
+int s2n_async_pkey_perform_stored_op(void)
+{
```

Comment:
Does this function need to exist anymore?

Reply:
No, I'll squash it